e2e/tests/testAllForms.spec.ts

```diff
@@ -37,10 +33,94 @@ test('All the forms should load on the patient chart page', async ({ page }) =>
   await expect(medicalHistoryForm?.includes('ប្រវត្តជំងឺ')).toBeTruthy();
   await expect(page.getByText('ប្រវត្តជំងឺ')).toBeVisible();
   await page.getByRole('button', { name: 'បិទ', exact: true }).click();
+  await homePage.deletePatient();
 });
 
-test.afterEach(async ({ page }) => {
+test('Location picker should search locations by entering a numeric value, a Khmer value and an English value from the location code', async ({ page }) => {
+  // setup
   const homePage = new HomePage(page);
-  await homePage.deletePatient();
+  await page.goto(`${process.env.E2E_BASE_URL}`);
+
+  // reply
+  await homePage.goToLocation();
+
+  // verify
+  let location = await page.locator('.cds--radio-button__label-text');
```

Comment:
This will return multiple HTML elements. Is this what you intended to achieve? Also is there no combination of locators to explicitly pick the underlying HTML element (the class locator is not quite explicit in itself since it could be used elsewhere).

Reply:
The locator is updated.